modules/tile-converter/src/converter-cli.ts

```diff
@@ -14,6 +14,8 @@ type TileConversionOptions = {
   /** Output folder. This folder will be created by converter if doesn't exist. It is relative to the converter path.
    * Default: "data" folder */
   output: string;
+  /** Keep created 3DNodeIndexDocument files on disk instead of memory. This option reduce memory usage but decelerates conversion speed */
+  instantNodesWriting: boolean;
```

Comment:
I would try to find a shorter option name. At a minium I would drop the `s` to make it a little easier to read/type
```suggestion
  instantNodeWriting: boolean;
```